readme.md

```diff
@@ -0,0 +1,15 @@
+# RangerQuiz
+
+RangerQuiz is an fun web app, through which you can enjoy quizzes of your favorite shows. In the end you will get to know the scores also. It consist of features as mentioned below.
+
+This web app is fully responsive.
+
+# Features
+
+1. [Home page](https://ranger-quiz.netlify.app/)
+2. [Quiz category page](https://ranger-quiz.netlify.app/quiz-category-page.html)
+3. [Rules page](https://ranger-quiz.netlify.app/rules-page.html)
+4. [Question page](https://ranger-quiz.netlify.app/question-page.html)
+5. [Result page](https://ranger-quiz.netlify.app/answer-page.html)
+
+[Animation](/gif/quiz-web-app.gif)
```

Comment:
`![Animation](/gif/quiz-web-app.gif)`  if you add this way then the gif will visible on the readme file

Reply:
okay